src/app/containers/PageHandlers/withError/index.test.jsx

```diff
@@ -9,14 +9,14 @@ describe('withError HOC', () => {
   describe('with error', () => {
     shouldShallowMatchSnapshot(
       `should return the errorMain component`,
-      <ErrorHOC error="error handling at its best" />,
+      <ErrorHOC error={new Error('This is an error.')} />,
     );
   });
 
   describe('with no error', () => {
     shouldShallowMatchSnapshot(
       `should return the passed in component`,
-      <ErrorHOC error={null} />,
+      <ErrorHOC error />,
```

Comment:
```suggestion
      <ErrorHOC />,
```
`<ErrorHOC error />` is the same as `<ErrorHOC error={true} />`

Reply:
good shout (snapshots were wrong as well)
![](https://media.giphy.com/media/7rTUToVCLdTxu/giphy.gif)